src/components/layouts/Sidebar.tsx

```diff
@@ -45,19 +68,24 @@ export default class Sidebar extends React.Component<{}> {
               </div>
             </Link>
             <div className="sidebar-bottom-links">
-            <Link to="/settings" style={{ textDecoration: "none" }}>
-              <div>
-                <div className="sidebar-inner">
-                  <img
-                    src="assets/icons/settings-icon.svg"
-                    className="sidebar-inner"
-                    alt="settings"
-                  />
-                  <div className="sidebar-head sidebar-inner">Settings</div>
+              <Link to="/settings" style={{ textDecoration: "none" }}>
+                <div>
+                  <div
+                    className={
+                      this.state.active[3] ? "sidebar-button-active" : ""
+                    }
+                    onClick={e => this.activeClassToggle(3)}
+                  >
+                    <img
+                      src="assets/icons/settings-icon.svg"
```

Comment:
Fix the alignment here

Setting button

![Screenshot from 2019-10-25 15-53-12](https://user-images.githubusercontent.com/30416891/67564088-d03a6900-f73f-11e9-9a42-11cb65d9727a.png)